datasets/gov/vstt/saugomu_ter_kadastras.csv

```diff
@@ -0,0 +1,716 @@
+,,,,,gat_kodas,integer,,,,2,open,,Gatvės kodas,
+,,,,,gat_pavadinimas,string,,,,2,open,,Gatvės pavadinimas,
+,,,,,mu_kod,integer,,,,2,open,,Miškų urėdijos kodas,
+,,,,,mu_pavad,string,,,,2,open,,Miškų urėdijos pavadinimas,
+,,,,,gir_kod,integer,,,,2,open,,Miškų girininkijos kodas,
+,,,,,gir_pavad,string,,,,2,open,,Miškų girininkijos pavadinimas,
+,,,,,kv_nr,integer,,,,2,open,,Miškų kvartalo numeris,
+,,,,,skl_nr,integer,,,,2,open,,Miškų sklypo numeris,
+,,,,,skl_geo,string,,,,2,open,,Geokodas,Unikalus miško sklypo geokodas susidedantis iš urėdijos girininkijos kvartalo sklypo numerio sklypo dalies kodų
+,,,,,gamt_paminkl,integer,,,,4,open,,Saugojamas gamtos paminklas,Ar saugomas objektas yra gamtos paminklas
+,,,,,,enum,,,0,,,,Nepaminklas,
+,,,,,,,,,1,,,,Paminklas,
+,,,,,gamt_pam_steig_ta,string,,,,4,open,,Teisės aktas,Paskelbimo gamtos paminklu teisinis pagrindas
+,,,,,gamt_pam_steig_data,date,D,,,4,open,,Paskelbimo gamtos paminklu data,
+,,,,,bendr_reglam,integer,,,,2,open,,Reglamentas,Bendrieji veiklos reglamentai nuoroda į teisės aktą
+,,,,,spec_reglam,string,,,,4,open,,Reglamentai,Specialieji / individualieji veiklos reglamentai
+,,,,,saltiniai,string,,,,2,open,,Duomenų  šaltiniai,Nurodomi moksliniai tyrimai ir publikacijos
+,,,,,buf_zona_apras,string,,,,2,open,,Buferinė zona,Duomenys apie valstybinio parko buferinę apsaugos zoną
+,,,,,pastabos,string,,,,2,open,,Pastabos,
+,,,,,id_vart,integer,,,,2,open,,Duomenų teikėjas, Institucija/-jos atsakinga/-os už priežiūrą ir monitoringą. Duomenų teikėjų klasifikatorius
+,,,,,steigejas_galiojantis,string,,,,2,open,,Steigėjas,Institucija kurios potvarkiu šiuo metu galioja saugoma teritorija arba jos riba
+,,,,,galiojantis,date,D,,,4,open,,Steigimo data,Šiuo metu galiojančio saugomos teritorijos steigimo teisės akto data
+,,,,,steig_ta_galiojantis,string,,,,4,open,,Teisinis statusas,Teisės aktas pagal kurį šiuo metu galioja gamtos paveldo objekto riba (vieta) reglamentai arba šis teisės aktas yra šiuo metu galiojantis gamtos paveldo objekto teisinis pagrindas
+,,,,,planavimo_dok,string,,,,2,open,,Planavimo dokumentas,Duomenys apie parengtus planavimo dokumentus
+,,,,,vieta,string,,,,2,open,,Vieta,Vieta aprašomoji informacija (savivaldybė seniūnija jei miškas – papildomai miškų urėdija girininkija miško kvartalas ir sklypas)
+,,,,,obj_status,integer,,,,2,open,,Objekto būsena,
+,,,,,koord,"geometry(point,4326)",m,,,4,open,,Taško koordinatės,
+,,,,,,,,,,,,,,
+,,,,BuferineApsaugosZona,,,id,,,4,,,Buferinės apsaugos zonos,
+,,,,,gkodas,string,,,,2,open,,InGIS geokodas,Saugomos teritorijos tipas
+,,,,,st_gkodas,string,,,,2,open,,Teritorijos kodas,Saugomos teritorijos kuriai priklauso buferinės apsaugos zona tipas – geokodas (visų saugomų teritorijos tipų pagal InGIS standartą klasifikatorius)
+,,,,,st_id,string,,,,2,open,,Identifikatorius,Jei gamtos paveldo objektas yra saugomoje teritorijoje pildomas saugomos teritorijos kurioje yra gamtos paveldo objektas identifikatorius
```

Comment:
Ar identifikatorius yra neunikalus ar nepatikimas? Ar jis ateina iš kt. registro ar IS?